src/molecules/cardDate/cardDate.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { Chip } from '../../atoms/chips/chip';
+import { ChipProps } from '../../atoms/chips/types';
+import { Text } from '../../atoms/texts/text';
+import { CardDateContainer } from './styles';
+import { CardDateProps } from './types';
+
+export const CardDate: FC<CardDateProps> = ({ state, date }) => {
+  const secondaryTypeBgColor: ChipProps['secondaryTypeBgColor'] = state === 'Upcoming' ? 'orange' : state === 'New' ? 'green' : 'red';
+  return (
+    <CardDateContainer>
+      <Chip secondaryTypeBgColor={secondaryTypeBgColor} type="secondary">{state}</Chip>
+      <Text size="L-Bold">{date}</Text>
```

Comment:
@ArinCantCode 18 font-size ? , can u double check with Michal about it, maybe she has different point of view already , if not lets stay like this.